source/dea_shutdown.rst

```diff
@@ -43,52 +52,131 @@ What is the first response?
+
+.. |deaa_on_pdf| replace:: PDF
+.. _deaa_on_pdf: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_61036_DEAA_ON.pdf
+
+.. |deaa_on_doc| replace:: DOC
+.. _deaa_on_doc: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_61036_DEAA_ON.doc
+
+.. |wmboot_hkp| replace:: ``SOP_ACIS_WARMBOOT_DEAHOUSEKEEPING``
+.. _wmboot_hkp: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_ACIS_WARMBOOT_DEAHOUSEKEEPING.pdf 
+
+.. |wmboot_hkp_pdf| replace:: PDF
+.. _wmboot_hkp_pdf: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_ACIS_WARMBOOT_DEAHOUSEKEEPING.pdf 
+
+.. |wmboot_hkp_doc| replace:: DOC
+.. _wmboot_hkp_doc: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_ACIS_WARMBOOT_DEAHOUSEKEEPING.doc
+		 
+.. |fptemp_121| replace:: ``SOT_SI_SET_ACIS_FP_TEMP_TO_M121C``
+.. _fptemp_121: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_SI_SET_ACIS_FP_TEMP_TO_M121C.pdf
+
+.. |fptemp_121_pdf| replace:: PDF
+.. _fptemp_121_pdf: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_SI_SET_ACIS_FP_TEMP_TO_M121C.pdf
+
+.. |fptemp_121_doc| replace:: DOC
+.. _fptemp_121_doc: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_SI_SET_ACIS_FP_TEMP_TO_M121C.doc
+
+.. |deaonssc| replace:: ``I_1_DEAA_ON.ssc``
+.. _deaonssc: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/products/ssc/I_1_DEAA_ON.ssc  
+ 
+.. |warmbootssc| replace:: ``I_1_WARMBOOT.ssc``
+.. _warmbootssc: https://occweb.cfa.harvard.edu/occweb/FOT/configuration/products/ssc/I_1_WARMBOOT.ssc  
+ 
 SOT Procedures
 ++++++++++++++
 
 * `Turn On DEA-A <http://cxc.cfa.harvard.edu/acis/cmd_seq/deaa_on.pdf>`_
 * `Warm Boot the Active ACIS BEP and Start DEA HK Run <http://cxc.cfa.harvard.edu/acis/cmd_seq/warmboot_hkp.pdf>`_
-* `Set Focal Plane Temperature to -119.7 C <http://cxc.cfa.harvard.edu/acis/cmd_seq/setfp_m121.pdf>`_
+* `Set Focal Plane Temperature to -121 C <http://cxc.cfa.harvard.edu/acis/cmd_seq/setfp_m121.pdf>`_
 
 FOT Procedures
 ++++++++++++++
 
-.. |deaa_on| replace:: ``SOP_61036_DEAA_ON``
-.. _deaa_on: http://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_61036_DEAA_ON.pdf
+* ``SOP_61036_DEAA_ON`` (|deaa_on_pdf|_) (|deaa_on_doc|_)
+* ``SOP_ACIS_WARMBOOT_DEAHOUSEKEEPING`` (|wmboot_hkp_pdf|_) (|warmoot_hkp_doc|_)
+* ``SOT_SI_SET_ACIS_FP_TEMP_TO_M121C`` (|fptemp_121_pdf|_) (|fptemp_121_doc|_)
 
-.. |wmboot_hkp| replace:: ``SOP_ACIS_WARMBOOT_DEAHOUSEKEEPING``
-.. _wmboot_hkp: http://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_ACIS_WARMBOOT_DEAHOUSEKEEPING.pdf
+FOT Scripts
++++++++++++
 
-.. |fptemp_121| replace:: ``SOT_SI_SET_ACIS_FP_TEMP_TO_M121C``
-.. _fptemp_121: http://occweb.cfa.harvard.edu/occweb/FOT/configuration/procedures/SOP/SOP_SI_SET_ACIS_FP_TEMP_TO_M121C.pdf
+* |deaonssc|_
+* |warmbootssc|_
+
+CLD Scripts

```

Comment:
Since there are no CLD Scripts for this one, maybe we just remove this subsection entirely.